README.md

```diff
@@ -114,6 +114,21 @@ It is possible to add an optional image (with alternative text) to a step.
 - ...
 ```
 
+### Notices in Bot Messages
+
+It is possible to display a notice to the user, on any given step. This is
+useful when, for example, you want to give feedback to an action.
+
+```yaml
+- step1:
+    notice-type: correct
+    notice-text: Ding, Ding! We have a winner!
+    messages: Nice shot.
+```
+
+`notice-type` is the CSS class associated with the notice. You can leverage
+this to style the notices any way you like.
```

Comment:
Regular users don't usually add custom CSS to their LMS, though. Can you mention that  we currently have built in styles for `correct` and `incorrect` notice types?